src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,15 @@
+package duke;
+
+/**
+ * Custom Exception for the project: duke.Duke
+ */
+public class DukeException extends Exception {
+    public DukeException() {
+        super();
+    }
+
+
+    public DukeException(String message) {
+        super(message);
+    }
+}
```

Comment:
I've noticed this is your only Exception class. I feel it would be more apt to have multiple Exception classes that extend from your DukeException, each with their own message. For example, InsufficientArgumentsException extends DukeException.